app/src/RoomClient.js

```diff
@@ -298,6 +298,16 @@ export default class RoomClient
 
 				switch (key)
 				{
+					case String.fromCharCode(37):
+					{
+						this._spotlights.setPrevAsSelected();
+						break;
+					}
+					case String.fromCharCode(39):
+					{
+						this._spotlights.setNextAsSelected();
+						break;
+					}
```

Comment:
Please add these new buttons to the help!

Reply:
Done!

Reply:
Thanks! 
"Browse participants into Spotlight"
Maybe "Move" instead of "Browse" would be more clear? (at least for me.) 
What do you think?

LGTM

Reply:
I was thinking about it, but I cannot find a good short description, as what you do is browse (back and forth) the passive participants and move one by one these that currently are not visible into the Spotlight.